### ISSUE TITLE
Compute jacobians in local world oriented frame

### ISSUE THREAD

**Issue by jviereck** [stack-of-tasks/pinocchio#773]
Hi folks,

at this point the jacobians can be computed in the world or local frame. I propose to add another reference frame that is the local_cartesian_oriented frame. This is useful when dealing with point contacts.

The code to compute this boils down to:

```
        jac = se3.getFrameJacobian(self.model, self.data, index, se3.ReferenceFrame.LOCAL)
        world_R_joint = se3.SE3(self.data.oMf[index].rotation, zero(3))
        return world_R_joint.action.dot(jac)
```

As this is a convenient method to have, I would like to add it to the pinocchio se3/RobotWrapper Jacobian's C++ API but would like to get feedback of adding a PR with this feature has changes to getting accepted.

Best,
Julian

**Comment by jcarpent** [comment]
This feature is mainly worthy when you want to track a velocity field expressed at the origin of the moving frame, but with quantities expressed in the world axis.
It can then be worthy to have it, but it will also means that we will lose the fondamental relations that:
Jworld_body_i = oMi.act(Jlocal_body_i)
Then, the documentation has to be pretty clear on this point.

**Comment by gabrielebndn** [comment]
I would love to see more support for the traditional Cartesian way of understanding frames.

**Comment by jcarpent** [comment]
@jviereck Are you still willing to implement this feature?

**Comment by jviereck** [comment]
@jcarpent, if someone else jumps on it, I would not mind. Otherwise I want to make a first few steps on Friday.

**Comment by jcarpent** [comment]
From my side, I may help by reviewing the code and providing some hints if needed.
Maybe @gabrielebndn would like to do it?

**Comment by gabrielebndn** [comment]
I like the idea and I approve of it fully, but I am quite busy with a lot of stuff at the moment :)
I have just a few recommendations:
- if you do it for the frames, do not forget doing it for the joints too
- I think the best way to do it would be adding another element to the `enum` called `ReferenceFrame` and take the new value into account inside the existing functions

**Comment by jcarpent** [comment]
Solved by #781
